ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/synccommittee/SignedContributionAndProofValidator.java

```diff
@@ -80,10 +80,13 @@ public SignedContributionAndProofValidator(
     final Optional<SyncCommitteeUtil> maybeSyncCommitteeUtil =
         spec.getSyncCommitteeUtil(contribution.getSlot());
     if (maybeSyncCommitteeUtil.isEmpty()) {
-      LOG.trace(
-          "Rejecting proof because the fork active at slot {} does not support sync committees",
-          contribution.getSlot());
-      return SafeFuture.completedFuture(REJECT);
+      final String failureReason =
+          String.format(
+              "Rejecting proof because the fork active at slot %s does not support sync committees",
+              contribution.getSlot());
+      LOG.trace(failureReason);
+      return SafeFuture.completedFuture(
+          InternalValidationResult.create(ValidationResultCode.REJECT, failureReason));
```

Comment:
I wonder if we should extract a private method for this so it becomes something like:

```
return failureResult("Rejecting proof blah blah %s", contribution.getSlot());
```

The private method would take `String message, String... args` and do the `String.format` call for you.